src/components/Tutorials/subComps/EditControls.jsx

```diff
@@ -201,6 +297,9 @@ const EditControls = ({
               >
                 <FileCopyIcon /> {isPublished ? "Unpublish" : "Publish"}
               </Button>
+              <Button disabled={!isPublished} onClick={() => history.push("/")} >
```

Comment:
What do you think about simplifying and omitting the "Finish" button?

Reply:
I have made necessary changes as per the feedback and made PR #1032
@shivareddy6 could you please review it?